integration_test/issuance_test.dart

```diff
@@ -21,89 +19,141 @@ void main() {
+      final emailTileFinder = find.byKey(const Key('irma-demo.sidn-pbdf.email_tile'));
+      await tester.scrollUntilVisible(emailTileFinder, 75);
+      await tester.tapAndSettle(emailTileFinder);
 
-      // Expect detail page
+      // Expect the detail screen
       expect(find.byType(CredentialsDetailScreen), findsOneWidget);
 
-      // Expect to find a card
-      final credentialCardFinder = find.byType(IrmaCredentialCard);
-      expect(credentialCardFinder, findsOneWidget);
-    }
+      // Expect that it has one card.
+      final credentialCardsFinder = find.byType(IrmaCredentialCard);
+      expect(credentialCardsFinder, findsOneWidget);
 
-    testWidgets(
-      'issue-municipality-en',
-      (tester) => testIssueMunicipality(tester, const Locale('en', 'EN')),
-    );
+      // Expect the card to have the correct content
+      final credentialCardFinder = credentialCardsFinder.first;
 
-    testWidgets(
-      'issue-municipality-nl',
-      (tester) => testIssueMunicipality(tester, const Locale('nl', 'NL')),
-    );
+      Future<void> evaluateIssuedEmailAddressCard() => evaluateCredentialCard(
+            tester,
+            credentialCardFinder,
+            credentialName: 'Demo Email address',
+            issuerName: 'Demo Privacy by Design Foundation via SIDN',
+            attributes: {
+              'Email address': 'test@example.com',
+              'Email domain name': 'example.com',
+            },
+          );
 
-    testWidgets('decline', (tester) async {
-      await pumpAndUnlockApp(tester, irmaBinding.repository);
+      await evaluateIssuedEmailAddressCard();
 
-      // Start an issuance session for email address and decline the offer.
-      await issueCredentials(
-        tester,
-        irmaBinding,
-        {
-          'irma-demo.sidn-pbdf.email.email': 'test@demo.com',
-          'irma-demo.sidn-pbdf.email.domain': 'demo.com',
-        },
-        declineOffer: true,
-      );
+      // Issue the same credential again.
+      await issueEmailAddress(tester, irmaBinding);
+      await tester.tapAndSettle(okButtonFinder);
 
-      // Go to data tab.
-      await tester.tapAndSettle(find.byKey(const Key('nav_button_data')));
-
-      // Verify that the email address has not been added.
-      final emailTileFinder = find.byKey(const Key('irma-demo.sidn-pbdf.email_tile'));
-      expect(emailTileFinder, findsNothing);
+      // Expect the detail screen with one card again
+      await tester.tapAndSettle(emailTileFinder);
+      expect(find.byType(CredentialsDetailScreen), findsOneWidget);
+      expect(credentialCardsFinder, findsOneWidget);
 
-      // Verify that no activity has been added.
-      await tester.tap(find.byKey(const Key('nav_button_activity')));
-      await tester.pump(const Duration(seconds: 1));
-      expect(find.text('There are no logged activities yet'), findsOneWidget);
+      // Expect the card to have the correct content again
+      await evaluateIssuedEmailAddressCard();
```

Comment:
I think it's also nice to check whether the new credential instance actually replaced the old credential instance on the data tab. There should not be two credentials on the data tab with `test@example.com`.

Reply:
I do check this on the line above: `expect(credentialCardsFinder, findsOneWidget)` So there should only be one `IrmaCredentialCard` on the `CredentialsDetailScreen` after reissuing

Reply:
Ah okay, I didn't notice that the data tab is still active after the second session.